main/lsp/requests/definition.cc

```diff
@@ -9,6 +11,26 @@ DefinitionTask::DefinitionTask(const LSPConfiguration &config, MessageId id,
                                unique_ptr<TextDocumentPositionParams> params)
     : LSPRequestTask(config, move(id), LSPMethod::TextDocumentDefinition), params(move(params)) {}
 
+core::Loc
+DefinitionTask::findRequireRelativeLoc(const core::GlobalState &gs,
+                                       const std::vector<std::unique_ptr<core::lsp::QueryResponse>> &responses) {
+    if (responses.size() < 2)
+        return core::Loc::none();
+
+    auto parentSendResp = move(responses[1])->isSend();
+    const bool isRequireRelative = parentSendResp && parentSendResp->callerSideName == core::Names::require_relative();
+    auto literal = move(responses[0])->isLiteral();
```

Comment:
Why are you `move`ing here? This is a weird side effect.

Reply:
I'll check, maybe that's a left over of something else I did previously